src/commands/aircraft/takeoffPerf.ts

```diff
@@ -3,12 +3,14 @@ import { CommandCategory } from '../../constants';
 import { makeEmbed, makeLines } from '../../lib/embed';
 
 const takeoffPerfEmbed = makeEmbed({
-    title: 'FlyByWire A32NX | Where is the takeoff calculator?',
-    description: makeLines(['Currently, Airbus does not publicly release takeoff performance data for the A320neo, which means it is difficult to create an accurate takeoff calculator.',
+    title: 'FlyByWire A32NX | Takeoff Performance Calculator',
+    description: makeLines([
+        'A Takeoff Performance Calculator is available on the **Developer** version of the A32NX. For more information, check out the [Takeoff Performance Calculator documentation](https://docs.flybywiresim.com/flypad-performance/).',
         '',
-        'While there are many online calculators available, these often use A320ceo data, and are not accurate.',
+        'For the **Stable** version, see the [V-Speeds Section](https://docs.flybywiresim.com/pilots-corner/beginner-guide/preparing-mcdu/#v-speeds) in our MCDU guide.',
```

Comment:
```suggestion
        'For the **Stable** branch, see the [V-Speeds Section](https://docs.flybywiresim.com/pilots-corner/beginner-guide/preparing-mcdu/#v-speeds) in our MCDU guide.',
```

Reply:
Would leave it as version to keep it in the language of a non Dev.